app/frontend/src/routes.ts

```diff
@@ -1,3 +1,7 @@
+/*
+Please make sure this file stays in sync with
+//app/backend/src/couchers/urls.py
+*/
```

Comment:
No, //docs/urls.md is the primary source of URL information that both frontend and backend should refer to.